.vscode/launch.json

```diff
@@ -71,7 +71,7 @@
                 "SKIP_SERVICE_VERSION_REQUIREMENTS": "1",
                 "PRINT_SQL": "1",
                 "REPLAY_EVENTS_NEW_CONSUMER_RATIO": "1.0",
-                "BILLING_SERVICE_URL": "https://billing.dev.posthog.dev"
+                "BILLING_SERVICE_URL": "http://localhost:8100" // "https://billing.dev.posthog.dev"
```

Comment:
👀 

this isn't secret thankfully but be very careful to not commit your env vars 😄